### ISSUE TITLE
2.1 final : unable to save/load stories

### ISSUE THREAD

**Issue by klembot** [klembot/twinejs#335]
Originally reported by: **Pierre K. (Bitbucket: [Pierre_K](https://bitbucket.org/Pierre_K), GitHub: Unknown)**

----------------------------------------

**My config**
I'm using ver. 2.1 linux 64 on the last Mint distro with french localisation

**My problem**
I have created manually a ~/Documents/Twine/Histoires folder as indicated in the release notes
I launch Twine and create a new story, and edit the first passage
The new story file is created in ~/Documents/Twine/Stories but when a close Twine, no more file remains in both Histoires or Stories folder

**What i have tried**
I have deleted .conf/Twine settings folder with not results.
I have tried to set Twine to english language back, no results to

----------------------------------------
- Bitbucket: https://bitbucket.org/klembot/twinejs/issue/335

**Comment by vinniec** [comment]
Yesterday I tried twine for the first time downloading version 2.2.1 64bit for linux from the site.
So I don't know if it's the right place to report a bug.
The problem is simple, if in my home the directory ~/Documents/Twine/ is not created, the software does not save the stories that are being edited with any report.
I had to create this directory by hand because there wasn't on my O.S. (ubuntu 17.10 Italian locallization).
I think it would be enough at least to create the directory automatically.
